recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
+  version: "{{ version }}"
+
+source:
+  fn: hdfs3-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdfs3/hdfs3-{{ version }}.tar.gz
+  md5: ed0810314af1f4c147af0c1d6ed8a266
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [not linux]
```

Comment:
What are the roadblocks to having this running on other platforms?  Is there interest in getting it running on other platforms?  It would be nice to add a comment here about why it is linux-only.